pkg/container/bitlpm/unsigned_test.go

```diff
@@ -177,9 +178,7 @@ func TestUnsignedUpsert(t *testing.T) {
 				sort.Slice(got, func(i, j int) bool {
 					return got[i].start < got[j].start
 				})
-				if !reflect.DeepEqual(got, tt.ranges[:i+1]) {
-					t.Fatalf("When updating an unsigned trie with the key-prefix %d/%d: got %+v, but expected %+v", pr.start, pr.prefix(), got, tt.ranges[:i+1])
-				}
+				assert.Equal(t, tt.ranges[:i+1], got, "When updating an unsigned trie with the key-prefix %d/%d", pr.start, pr.prefix())
```

Comment:
nit: this should be `require.Equal` since the test previously fatal'd on failure